.github/workflows/_build.yml

```diff
@@ -11,7 +11,7 @@ defaults:
 env:
   flavour: oss
   config: debug
-  api-level: 24
+  api-level: 26
```

Comment:
switched to a newer api level to see if that stabilises things. unfortunately this means all the snapshots change as the font rendering is slightly different

Reply:
Is it a good and ok way for font rednering or is there something that needs to be fixed?

Reply:
nope it's just that the rendering changed in this OS version, so this is expected